engine/language-server/src/test/scala/org/enso/languageserver/websocket/json/VcsManagerTest.scala

```diff
@@ -1491,7 +1492,11 @@ class VcsManagerTest extends BaseServerTest with RetrySpec with FlakySpec {
 
   def commit(root: File, msg: String): RevCommit = {
     val jgit = new JGit(repository(root.toPath))
-    jgit.commit.setMessage(msg).setAuthor("Enso VCS", "vcs@enso.io").call()
+    jgit.commit
+      .setMessage(msg)
+      .setAuthor("Enso VCS", "vcs@enso.io")
```

Comment:
Why `enso.io`, not `enso.org`?

Reply:
Thanks. I somehow managed to convince myself Enso is `io` not `org`. Let me check my ID to verify that I know what is my name...